tests/test_indexer.py

```diff
@@ -286,7 +287,8 @@ def subscribe_for_notification(self, es_query, callback_url):
             requests.codes.created,
             json_request_body=dict(
                 es_query=es_query,
-                callback_url=callback_url),
+                callback_url=callback_url,
+                hmac_secret_key=PostTestHandler.hmac_secret_key),
```

Comment:
Do we expect that in production all subscriptions will always use HMAC?
If so, then I'm OK with only testing this case.
Otherwise, I think there should be unit tests for both the case where HMAC is configured in the subscription, and the case in which it is not configured in the subscription.

Reply:
@Bento007/I can help by adding the test for the case where HMAC is not configured, if you agree this case should be tested.

Reply:
Yes, I need to add a test for that.

Reply:
Test added.